ruby-challenges.rb

```diff
@@ -1,32 +1,105 @@
 # ASSESSMENT 5: Ruby Coding Practical Questions
-# MINASWAN
+# MINASWAN = "This is the way..." - Paz Vizla (RIP)
 
 # --------------------1) Create a method that takes in an array of words and a single letter and returns an array of all the words containing that particular letter. Use the test variables provided.
 
+# Pseudo code
+    # Use the select method in Ruby, which returns a new array containing elements that satisfy a given condition.
+    # Pass a block that checks if each word in the words array includes the given letter, case-insensitive.
+    # Use the downcase method to ensure that the comparison is case-insensitive.
+
 beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
 
 letter_o = 'o'
 # Expected output: ['coffee', 'soda water']
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+def words_containing_letter(words, letter)
+    filtered_words = words.select { |word| word.downcase.include?(letter.downcase) }
+    return filtered_words
+end
+
```

Comment:
Don't forget Ruby has implicit return! If you wanted to use the keyword `return`, you could just throw it in place of the variable!

Reply:
Thank you for catching that. I was up late and when I got to the Ruby challenges, it was past 2am, I think 🙈... I will keep in mind.